lessonbuilder/components/src/java/org/sakaiproject/lessonbuildertool/model/SimplePageToolDaoImpl.java

```diff
@@ -651,12 +651,15 @@ public boolean saveItem(Object o, List<String>elist, String nowriteerr, boolean
 			if (o instanceof SimplePageItem) {
 				SimplePageItem item = (SimplePageItem)o;
 				EventTrackingService.post(EventTrackingService.newEvent(LessonBuilderEvents.CREATE, "/lessonbuilder/item/" + item.getId(), true));
+				EventTrackingService.post(EventTrackingService.newEvent(LessonBuilderEvents.ITEM_CREATE, "/lessonbuilder/item/" + item.getId(), true));
```

Comment:
So we're going to post two events here from the same action? Or are you planning on deleting the previous event?